.circleci/config.yml

```diff
@@ -781,6 +781,7 @@ workflows:
             - Build (PR)
       - playwright-functional-tests:
           name: Functional Tests - Playwright (PR)
+          resource_class: xlarge
```

Comment:
There was at least one consistently failing test in CI (but not local) that's been fixed by this.  I decided to try this after noticing both CPU and RAM were pegged at 100%.

Reply:
> Can we file a follow up for this?

I would like to but I don't think this PR could pass CI without the change.  I was seeing consistently failing tests that this change immediately fixed.  I'll try reverting and running it through CI again but I'm not particularly hopeful.

Reply:
Oh, no need to revert! By file a follow up, I just mean a separate issue to investigate this. I'd be interested in profiling the code and seeing if we can tell what's hogging memory. I think for now it makes sense to increase the resource class 👍 . 

If it turns out we do have a memory issue or leak, I don't feel like it land on this PR's shoulders to fix it anyways.

Reply:
errr it took a few tries, but now the Playwright job is passing on `large`.  I'll leave it alone then.  Thanks for calling it out.  🙇

Reply:
😬 
![image](https://user-images.githubusercontent.com/198055/225396523-46ce3fb6-9e44-4459-becc-f9bc1b60990b.png)

Also multiple ``The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process.`` in the payments-react log.

Reply:
Although the payments related failure was actually on Parallel Run 5, which never quite reach 100% RAM usage but stayed 100% CPU for a while...
![image](https://user-images.githubusercontent.com/198055/225398426-fc14f9d5-73c6-4916-81d4-2afd7860d814.png)